_query-dsl/term/term.md

```diff
@@ -42,6 +42,9 @@ GET shakespeare/_search
 ```
 {% include copy-curl.html %}
 
+In OpenSearch 2.x and previous versions, complexity may scale exponentially with the number of characters, consuming heap memory and impacting performance. Case-insensitive searches should be avoided by using a [lowercase token filter]({{site.url}}{{site.baseurl}}/analyzers/token-filters/lowercase/) on the indexed field's analyzer, and then using lowercase query terms instead of relying on case-insensitive searches.
```

Comment:
```suggestion
In OpenSearch 2.x and earlier, complexity can increase exponentially with the number of characters, leading to high heap memory usage and reduced performance. To avoid this, do not use case-insensitive searches. Instead, apply a [lowercase token filter]({{site.url}}{{site.baseurl}}/analyzers/token-filters/lowercase/) in the indexed field's analyzer and use lowercase query terms.
```

Reply:
went to commit this but it's already there. Thanks!